tests/performance/parallel_algorithms/local/benchmark_unique_copy.cpp

```diff
@@ -29,7 +29,7 @@
 struct random_fill
 {
     random_fill(std::size_t random_range)
-        : gen(std::rand()),
+        : gen(std::random_device{}()),
```

Comment:
There is a problem that ```gen``` generates different numbers even if same 'seed' is used in program option.